File: absoluteValue.cpp
#include <iostream>

int absoluteValue(int input) {
	if (input >= 0) {
		return input;
	} else {
		return 0 - input;
	}
}

int main() {
	std::cout << "Enter your integer: ";
	int Integer;
	std::cin >> Integer;
	std::cout << absoluteValue(Integer);
	return 0;
}

//comment

File: absoluteValue.js
function absoluteValue(input) {
	if (input >= 0) {
		return input;
	} else {
		return 0 - input;
	}
}

let Integer;
const readline = require('readline').createInterface({
	input: process.stdin,
	output: process.stdout
});

readline.question('Enter your integer: ', Integer => {
	console.log (`${absoluteValue(Integer)}`);
	readline.close();
});